packages/flutter_tools/test/general.shard/dart_plugin_test.dart

```diff
@@ -139,7 +139,6 @@ void main() {
               appDependencies: directDependencies,
             ),
           ],
-          throwOnPluginPubspecError: false,
```

Comment:
Note that this test failed before, when setting `throwOnPluginPubspecError: true`, which was a wrong behavior.